shared-data/default-theme/html/jsapi/search/ui.js

```diff
@@ -79,6 +75,14 @@ Mailpile.results_list = function() {
 };
 
 
+Mailpile.display_keybindings = function(elem) {
+  Mailpile.API.with_template("modal-display-keybindings", function(modal) {
```

Comment:
Good catch. I agree the modal should follow the Highlander-Pattern preventing subsequent calls modals while one is open.